pyproject.toml

```diff
@@ -48,9 +48,6 @@ Homepage = "https://github.com/bwhmather/ssort"
 force-exclude = 'test_data/samples/*'
 line_length = 79
 
-[tool.distutils.bdist_wheel]
-universal = 1
-
```

Comment:
This seems to enable building a python 2 compatible wheel, which we don't want as we can't support python2.